scripts/word-to-dtbook/src/main/resources/xml/word-to-dtbook.xpl

```diff
@@ -0,0 +1,232 @@
+				</choice>
+			</px:type>
+		</p:pipeinfo>
+	</p:option>
+	<p:option name="DPI" select="96" cx:as="xs:integer">
+		<p:documentation xmlns="http://www.w3.org/1999/xhtml">
+			<h2 px:role="name">Image resampling value</h2>
+			<p px:role="desc">Image resampling targeted resolution in dpi (dot-per-inch)</p>
+		</p:documentation>
+	</p:option>
+	<p:option name="CharacterStyles" select="'false'" px:type="boolean">
+		<p:documentation xmlns="http://www.w3.org/1999/xhtml">
+			<h2 px:role="name">Translate character styles</h2>
+		</p:documentation>
+	</p:option>
+	<p:option name="FootnotesPosition" select="'end'">
+		<p:documentation xmlns="http://www.w3.org/1999/xhtml">
+			<h2 px:role="name">Footnotes position</h2>
+			<p px:role="desc">Footnotes position in content</p>
+		</p:documentation>
+		<p:pipeinfo>
+			<px:type>
+				<choice xmlns:a="http://relaxng.org/ns/compatibility/annotations/1.0">
+					<value>inline</value>
+					<a:documentation xml:lang="en">Inline note in content (after the paragraph containing its first reference)</a:documentation>
+					<value>end</value>
+					<a:documentation xml:lang="en">Put notes at the end of a level defined in footnotes insertion level</a:documentation>
+					<value>page</value>
+					<a:documentation xml:lang="en">Put the notes near the page break</a:documentation>
+				</choice>
+			</px:type>
+		</p:pipeinfo>
+	</p:option>
+	<p:option name="FootnotesLevel" select="0" cx:as="xs:integer">
+		<p:documentation xmlns="http://www.w3.org/1999/xhtml">
+			<h2 px:role="name">Footnotes insertion level</h2>
+			<p px:role="desc">Lowest level into which notes are inserted in content.
+			0 means the footnotes will be inserted as close as possible of its first call.</p>
+		</p:documentation>
+	</p:option>
+	<p:option name="FootnotesNumbering" cx:as="xs:string" select="'none'">
+		<p:documentation xmlns="http://www.w3.org/1999/xhtml">
+			<h2 px:role="name">Footnotes numbering</h2>
+			<p px:role="desc">Customize footnotes numbering</p>
+		</p:documentation>
+		<p:pipeinfo>
+			<px:type>
+				<choice xmlns:a="http://relaxng.org/ns/compatibility/annotations/1.0">
+					<value>none</value>
+					<a:documentation xml:lang="en">Disable note numbering</a:documentation>
+					<value>word</value>
+					<a:documentation xml:lang="en">Use original word numbering</a:documentation>
+					<value>number</value>
+					<a:documentation xml:lang="en">Use custom numbering, starting from the footnotes start value</a:documentation>
+				</choice>
+			</px:type>
+		</p:pipeinfo>
+	</p:option>
+	<p:option name="FootnotesStartValue" cx:as="xs:integer" select="1">
+		<p:documentation xmlns="http://www.w3.org/1999/xhtml">
+			<h2 px:role="name">Footnotes starting value</h2>
+			<p px:role="desc">If footnotes numbering is required, start the notes numbering process from this value</p>
+		</p:documentation>
+	</p:option>
+	<p:option name="FootnotesNumberingPrefix" cx:as="xs:string?" select="''">
```

Comment:
Note that `cx:as` on XProc script definitions is currently not supported. Actually I did happen to add support for it recently, but it's still in my development branch, and not everything is supported, e.g. `xs:string?` isn't.

What is the purpose of using `xs:string?`? What happens if you pass the empty sequence?

Reply:
I've removed the `cx:as="xs:string?"` attributes for now...